### ISSUE TITLE
Editor gets stuck

### ISSUE THREAD

**Issue by writer12** [olivierkes/manuskript#769]
Whenever I try to type, the cursor constantly gets stuck. I constantly have to stop typing and wait for it to catch up.

**Comment by obw** [comment]
Manuskript have some performance Issues, but this look like something new.

Can you give us some more informations:

Use Menu->Help->About to get the different Version of Libraries which are used by Manuskript, you can use Mark them with the Mouse and use cut&paste. Also on which OS. (Perhaps some little Info about the Hardware)

**Comment by writer12** [comment]
I managed to fix it by turning of spell check. I do know that some other people have been reporting this issue whenever they write scenes over 3000 words. I think it will be more helpfull to ask them, but thank you for trying to help. I really appreciate it.

**Comment by obw** [comment]
I have Scenes with more then 3000 Words (Seldom but it's happens), the Spell-check is running and no Problem. Eaven with a 5000 Word Scene Monster...

OS: Linux (Kubuntu 18LTS)
Python: 3.6.9
PyQt: 5.13.1
Qt: 5.13.0

16GB RAM
Prozessor: AMD Ryzen 3 2200G

Even my Lab, 8G RAM and a Intel Mobile with 1600 has no such issues.
Oh, and I use German dictionaries.

Also I use the latest development-branch.

One last Question, are you using Images in your Text? This could make some problems, see #729

**Comment by writer12** [comment]
No, I'm not using Images. But I do have a different version from you, so maybe that is the issue.

**Comment by TheJackiMonster** [comment]
The performance has to be addressed for sure. ^^'
